src/main/java/DAOInterfaces/CommonUserDAOInterface.java

```diff
@@ -0,0 +1,23 @@
+package DAOInterfaces;
+
+import entity.CommonRole;
+import entity.CommonUser;
+import entity.CommonUserFace;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface CommonUserDAOInterface {
+
+    // create a CommonUser
+    public void createCommonUser(CommonUser user);
+
+    // update a CommonUser
+    public void updateCommonUser(CommonUser user);
+
+    // get all CommonUsers
+    public List<CommonUser> getAllCommonUsers();
+
+    // get one CommonUser
+    public CommonUser getCommonUser(UUID id);
```

Comment:
The input here should be Integer instead of UUID. We have UUID for Organization id and Integer for User id

Reply:
Thanks for the correction!